### DISCUSSION TITLE
Welcome to FreedomGPT Discussions!

### DISCUSSION THREAD

**Discussion by ohmguru** [ohmplatform/FreedomGPT#17] [Announcements]
## 👋 Welcome!
  FreedomGPT is using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by austin-365** [comment]
hi, here, I am wondering how to run the freedomGPT in linux? is that possible?

**Reply by gabrielklein** [reply]
Probably :) 

Install the libraries.
Just tell me the one you miss because I already have plenty of machine learning stuff installed...

sudo npm install --global yarn


git clone https://github.com/ohmplatform/FreedomGPT.git
mv FreedomeGPT FreedomGPTBin (Otherwise you will have a surprise...)

or (not sure of the correct one...) 

git clone https://github.com/ohmplatform/freedom-gpt-electron-app.git

cd FreedomGPTBin or cd freedom-gpt-electron-app.git
yarn install
yarn package
yarn start

I see the application screen. It starts to download the set.
Kill it and start it again.

But ... I don't get any answer when I try to use it. (trying to debug.)

**Reply by austin-365** [reply]
good advices! it works fine, util it fails with "Missing X server or $DISPLAY". i realized i am running this on a server without a screen. Any advices about how to run it in linux just with cli? Thank you~

**Reply by gabrielklein** [reply]
Yes I have first tried on my server too :) (I have a Nvidia 3070 for machine learning on it). 

When you ssh, try to add -X -Y

ssh -X -Y 1.2.3.4

You should be able to start the UI, but it doesn't answer too...

**Reply by raph-abdul** [reply]
STAY AWAY FROM FREEDOMGPT. THEY LEVERAGE YOUR COMPUTER POWER INSTALLING XMRIG TO MINE MONERO. CHECK THE LOGS OF YOUR COMPUTERS. FNT IS WORTH NOTHING. THEY PROFIT THOUSANDS OF DOLLARS EVERYDAY USING YOUR COMPUTER POWER AND GIVE YOU  THAT STUPID FUCKING FNT! I'VE FIGURED IT OUT AND EXPOSED THEM. OPEN AI IS BEHIND THIS.

**Comment by Mega493** [comment]
how to train the freedomgpt with your own models and datasets, i want to combine multiple models like gpt2 and bert-base-uncased for this AI to give better responses and use multiple datasets for talking to the AI, wil there be tab in the ui to give you the option to load and train these models and datasets, is it also possibly to ask the user to allocate system resources to the ai ? so the use the gpu and cpu if you can.

**Reply by raph-abdul** [reply]
STAY AWAY FROM FREEDOMGPT. THEY LEVERAGE YOUR COMPUTER POWER INSTALLING XMRIG TO MINE MONERO. CHECK THE LOGS OF YOUR COMPUTERS. FNT IS WORTH NOTHING. THEY PROFIT THOUSANDS OF DOLLARS EVERYDAY USING YOUR COMPUTER POWER AND GIVE YOU  THAT STUPID FUCKING FNT! I'VE FIGURED IT OUT AND EXPOSED THEM. OPEN AI IS BEHIND THIS.

**Comment by aaapple0** [comment]
I want to know why I use the question function after downloading, it keeps showing circles and then no results come out.

**Reply by raph-abdul** [reply]
STAY AWAY FROM FREEDOMGPT. THEY LEVERAGE YOUR COMPUTER POWER INSTALLING XMRIG TO MINE MONERO. CHECK THE LOGS OF YOUR COMPUTERS. FNT IS WORTH NOTHING. THEY PROFIT THOUSANDS OF DOLLARS EVERYDAY USING YOUR COMPUTER POWER AND GIVE YOU  THAT STUPID FUCKING FNT! I'VE FIGURED IT OUT AND EXPOSED THEM. OPEN AI IS BEHIND THIS.
